package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "firestore-m3",
+  "name": "@now-ims/ember-jetfuel",
```

Comment:
I'm not comfortable with moving this to the `@now-ims` scope, I'd rather this stay a top level package for the time being, or perhaps a new top level scope be created. I also don't really like the idea of changing the name to `ember-jetfuel`, the current name is pretty straightforward and describes what the project is. Introducing a new name just makes it less clear what the project is doing IMO. If it were wrapping and adding fundamentally new functionality, I think that would be fine.

Reply:
100% agree. let us modify and will push a new PR